.github/workflows/sandbox-creating.yml

```
@@ -158,34 +177,20 @@
-            --name "sandbox-creation" \
-            --variables \
-              "name=BRANCH_NAME,value=${BRANCH_NAME}" \
-              "name=PR_NUMBER,value=${PR_NUMBER}" \
-              "name=IS_PULL_REQUEST,value=1"; then
-            echo "::error::Failed to start pipeline execution"
-            exit 1
-          fi
+      - name: Start Pipeline Execution
+        run: |
+          aws codepipeline start-pipeline-execution --name "sandbox-creation" \
+            --variables "name=BRANCH_NAME,value=${BRANCH_NAME}" "name=PR_NUMBER,value=${PR_NUMBER}" "name=IS_PULL_REQUEST,value=1"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)